scripts/ci/install-minikube.sh

```diff
@@ -1,98 +1,122 @@
 #!/bin/bash
 
-set -v
-
 OS=linux
 ARCH=amd64
-TARGET_DIR="/usr/bin"
+TARGET_DIR=/usr/bin
 
-MINIKUBE_VERSION="v0.24.1"
+MINIKUBE_VERSION="v0.25.2"
```

Comment:
why not using the latest one v0.26.1 ?

Reply:
Had some issues with 0.26.x, which may be related to the "none" driver, in any case the 0.25.2 is compatible with helm setup so I prefer to stick with 0.25.2

Reply:
ok